stdlib/universe/table_fns_test.go

```diff
@@ -143,23 +73,87 @@ func TestTableFind_Call(t *testing.T) {
+			want: func() flux.Table {
+				want := &executetest.Table{
+					KeyCols: []string{"_measurement", "user"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+						{Label: "_measurement", Type: flux.TString},
+						{Label: "user", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{mustParseTime(`2018-05-22T19:53:26.000000000Z`), 0.0, "CPU", "user1"},
+						{mustParseTime(`2018-05-22T19:53:36.000000000Z`), 1.0, "CPU", "user1"},
+					},
+				}
+				want.Normalize()
+				return want
+			}(),
+			fn: `f = (key) => key.user == "user1" and key._measurement == "CPU"`,
 		},
 		{
 			name: "exactly one match 2", // second table
-			want: tables[1],
-			fn:   `f = (key) => key.user == "user2"`,
+			want: func() flux.Table {
+				want := &executetest.Table{
+					KeyCols: []string{"_measurement", "user"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+						{Label: "_measurement", Type: flux.TString},
+						{Label: "user", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{mustParseTime(`2018-05-22T19:53:26.000000000Z`), 4.0, "CPU", "user2"},
+						{mustParseTime(`2018-05-22T19:53:36.000000000Z`), 20.0, "CPU", "user2"},
+						{mustParseTime(`2018-05-22T19:53:46.000000000Z`), 7.0, "CPU", "user2"},
+					},
+				}
+				want.Normalize()
+				return want
+			}(),
+			fn: `f = (key) => key.user == "user2"`,
 		},
 		{
 			name: "exactly one match 3", // third table
-			want: tables[2],
-			fn:   `f = (key) => key.user == "user1" and key._measurement == "RAM"`,
+			want: func() flux.Table {
+				want := &executetest.Table{
+					KeyCols: []string{"_measurement", "user"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+						{Label: "_measurement", Type: flux.TString},
+						{Label: "user", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{mustParseTime(`2018-05-22T19:53:26.000000000Z`), 1.0, "RAM", "user1"},
+					},
+				}
+				want.Normalize()
+				return want
+			}(),
+			fn: `f = (key) => key.user == "user1" and key._measurement == "RAM"`,
 		},
 		{
 			name: "multiple match", // first and third
-			want: tables[0],
-			fn:   `f = (key) => key.user == "user1"`,
+			want: func() flux.Table {
+				want := &executetest.Table{
+					KeyCols: []string{"_measurement", "user"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+						{Label: "_measurement", Type: flux.TString},
+						{Label: "user", Type: flux.TString},
+					},
+					Data: [][]interface{}{
+						{mustParseTime(`2018-05-22T19:53:26.000000000Z`), 0.0, "CPU", "user1"},
+						{mustParseTime(`2018-05-22T19:53:36.000000000Z`), 1.0, "CPU", "user1"},
+					},
+				}
+				want.Normalize()
+				return want
+			}(),
+			fn: `f = (key) => key.user == "user1"`,
```

Comment:
It seems like you're just "rearranging" the code in this file. Is that all that's going on?

Reply:
The previous code executed the query once and stored the table results to be compared with an init function and this seemed like too much. I had to change the functions to execute the csv as a prelude rather than inject a table object into the script to avoid two calls to the interpreter so I figured I might as well make these tests more similar to other ones that do the same thing.